tests/test_token.py

```diff
@@ -41,7 +46,7 @@ def test_extract_encrypted_token_sensitive_content_no_token(client):
 def test_encrypted_and_signed_token(client):
     username = "researchuser"
     expires_in = datetime.timedelta(minutes=1)
-    expiry_datetime = datetime.datetime.utcnow() + expires_in
+    expiry_datetime = dds_web.utils.current_time() + expires_in
     additional_claim = {"iss": "DDS"}
```

Comment:
I now that you didn't change that and it is only a test, but since I just noticed: 

Would DDS be a valid `iss` claim? Shouldn't that be [a URI in URN format](https://en.wikipedia.org/wiki/Uniform_Resource_Name)?

Reply:
This is a bogus test string. One can understand that it is possible to include such additional claims by either looking at a test like this or reading the docstring written for the function.